targets/csharp/source/PlayFabSDK/source/Qos/QosRegionResult.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace PlayFab.QoS
+{
+    using MultiplayerModels;
+
+    public class QosRegionResult
+    {
+        public AzureRegion Region { get; set; }
```

Comment:
Unless you have an important reason to use properties, please use fields in customer facing code.
Fields are 1000x more performant than properties.

Reply:
Would you point me any reference about this performance comparison. I have read several posts about this, and they all say that auto property performs the same as field. [https://stackoverflow.com/questions/632831/why-are-public-fields-faster-than-properties](https://stackoverflow.com/questions/632831/why-are-public-fields-faster-than-properties) 

---
In reply to: [288687997](https://github.com/PlayFab/SDKGenerator/pull/535#discussion_r288687997) [](ancestors = 288687997)

Reply:
Change them to properties.

---
In reply to: [289988178](https://github.com/PlayFab/SDKGenerator/pull/535#discussion_r289988178) [](ancestors = 289988178,288687997)